api/client/logs.go

```diff
@@ -16,7 +16,7 @@ func (cli *DockerCli) CmdLogs(args ...string) error {
 		cmd    = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
 		follow = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
 		times  = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
-		tail   = cmd.String([]string{"-tail"}, "all", "Number of lines to show from the end of the logs")
+		tail   = cmd.String([]string{"-tail"}, "latest", "Number of lines to show from the end of the logs")
```

Comment:
Sorry, missed this comment. Will have a look later and see if I can make a suggestion. The 80 chars limit can be a bit limiting; true